### DISCUSSION TITLE
A warning from the other side: Filecoin and the R.I.C.O. Statute - Discussions censored multiple times on Filecoin/IPFS Discord

### DISCUSSION THREAD

**Discussion by mrhempman69** [filecoin-project/community#541] [General]
Filecoin and the R.I.C.O. Statute provides a new perspective on Filecoin from the view of a potential law enforcement officer. Actual video evidence, drawn from over 44 hours of footage, of Filecoin developers contradicting themselves is used to back its points. After the hierarchy of Filecoin is explained (not simply the Filecoin “protocol”), three examples provide details on cases where the RICO statute has been applied that may also apply to Filecoin. This video is a warning for Filecoin investors, including storage providers.

This 15-minute YouTube video has resulted in multiple account bans from the Filecoin/IPFS discord. The arguments from the sides who found value in this video were removed, while arguments that were pro-Filecoin were kept. Multiple accounts were banned for arguing in “bad faith”, “trolling”, “circular”, “straw-man”, “harassing”, “spam”, “being annoying”, despite using actual video evidence to back up their points. This biased censorship against something with so much work put into it clearly goes against Filecoin's core values, and is another attempt by these developers to deceive money out of you (as the video evidence also confirms).

Please only watch and comment if you have the 15-minutes to. If you have to wait until you’re eating or something like that, then save it for then.

**As this is a Filecoin discussion board, reading the forum replies about this video won’t give you an objective view of the video. There are fake accounts who take posts like this and spin them as quickly as possible without honestly watching the content. YOU have to actually watch the video and decide for yourself whether the cases where the RICO statute has been applied will also apply to Filecoin, in the context of the overrall Filecoin hierarchy.**

**Here is the video:** [Filecoin and the RICO Statute (Filecoin Explained) - YouTube](https://youtu.be/Z66xZQO0UOQ)

**Comment by IdaParks** [comment]
When it comes to discussions about any cryptocurrency or investment. it's always crucial to conduct thorough research and be critical of the
information presented. If there are conflicting opinions. it's essential to evaluate the evidence and form your own informed judgment.
